src/Lib/Managers/ItemsManager.vala

```diff
@@ -501,6 +502,8 @@ public class Akira.Lib.Managers.ItemsManager : Object {
         }
 
         compile_model ();
+        view_canvas.pause_redraw = false;
+        view_canvas.request_redraw (view_canvas.get_bounds ());
```

Comment:
@mbfraga I had to add this in order for the created Artboard to be rendered.
I guess this is a regression from #695